messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/ExternalBrokerConfigurationService.java

```diff
@@ -64,31 +65,31 @@ public void stop(StopContext context) {
     }
 
     public Map<String, TransportConfiguration> getConnectors() {
-        return connectors;
+        return Collections.unmodifiableMap(connectors);
```

Comment:
What's the relation between this change and WFLY-19339?

Reply:
nothing